pilot.go

```diff
@@ -160,6 +108,63 @@ func initAutoPilot(svr *server, cfg *autoPilotConfig) (*autopilot.Agent, error)
 		},
 		Graph:           autopilot.ChannelGraphFromDatabase(svr.chanDB.ChannelGraph()),
 		MaxPendingOpens: 10,
+		ConnectToPeer: func(target *btcec.PublicKey, addrs []net.Addr) error {
+			// We can't establish a channel if no addresses were
+			// provided for the peer.
+			if len(addrs) == 0 {
+				return errors.New("no addresses specified")
+			}
+
+			// First, we'll check if we're already connected to the
+			// target peer. If not, then we'll need to establish a
+			// connection.
+			if _, err := svr.FindPeer(target); err != nil {
```

Comment:
return early instead if already connected :)

Reply:
Yep, would reduce the vertical indentation space here.